manifests/generated/vm-resource.yaml

```diff
@@ -7,6 +7,17 @@ metadata:
     kubevirt.io: ""
   name: virtualmachines.kubevirt.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
+    type: date
+  - JSONPath: .spec.running
+    name: Running
+    type: boolean
+  - JSONPath: .spec.volumes[0].name
```

Comment:
We also support setting the boot-order and I am not really a fan of relying on the implicit knowledge that you have to order your volume list to get a reasonable output.